microservices/scraper-thaiNGO/scraper.py

```diff
@@ -0,0 +1,71 @@
+from bs4 import BeautifulSoup
+import requests
+from urllib.request import urlopen
+import csv
+
+baseurl = "http://wiki.p2pfoundation.net/NGOs_in_Thailand"
+
+"""
+list of dictionaries to store ngo information
+ngos = [{
```

Comment:
Nit: Can we remove the example and just leave the comment of "list of dictionaries to store ngo information" since the keys can be inferred from ngos_store_keys. This also saves us from having to update the comment if we change it.